draft-ietf-scitt-architecture.md

```diff
@@ -149,17 +149,17 @@ Issuers can register their Signed Statements on any Transparency Service, with t
 This document describes the scalable, flexible, and decentralized SCITT architecture.
 Its goal is to enhance auditability and accountability across supply chains.
 
-In supply chains, artifacts travel down the chain until they are eventually consumed by someone.
-Consumers like to have information about the artifacts that they consume.
+In supply chains, artifacts travel down the chain until they are no longer usable for one or more parties.
```

Comment:
I would rephrase this.

Downstream artifacts, are built from upstream artifacts.

Traceability and quality control in supply chains becomes more difficult, as the number of components and contributors to an artifact increases.

Reply:
I was trying to stay in scope, I will admit it is awkward. I will rephrase given your feedback.

Reply:
I made another attempt in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/207/commits/11b19c34e128846112b37aac52e0d5d7b06b8514. Feel free to re-open if you feel it is not resolved.